docs/src/content/hardhat-runner/docs/guides/project-setup.md

```diff
@@ -15,38 +15,59 @@ $ npx hardhat
 888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
 888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
 
-Welcome to Hardhat v2.0.0
+Welcome to Hardhat v2.10.0
 
 ? What do you want to do? …
-❯ Create a sample project
-  Create an advanced sample project
-  Create an advanced sample project that uses TypeScript
+▸ Create a JavaScript project
+  Create a TypeScript project
   Create an empty hardhat.config.js
   Quit
 ```
 
 If you select _Create an empty hardhat.config.js_, Hardhat will create a `hardhat.config.js` like the following:
 
 ```js
-/**
- * @type import('hardhat/config').HardhatUserConfig
- */
+/** @type import('hardhat/config').HardhatUserConfig */
 module.exports = {
-  solidity: "0.7.3",
+  solidity: "0.8.9",
 };
 ```
 
-Then, you need to install Hardhat with:
+Then, you need to install Hardhat. We recommend using npm 7 or later:
+
+::::tabsgroup{options="npm 7+,npm 6,yarn"}
+
+:::tab{value="npm 7+"}
+
+```
+npm install --save-dev hardhat
+```
+
+:::
+
+:::tab{value="npm 6"}
 
 ```
 npm install --save-dev hardhat
 ```
 
+:::
+
+:::tab{value="yarn"}
+
+```
+yarn add --dev hardhat
+```
+
+:::
+
+::::
+
 And this is enough to run Hardhat using a default project structure.
 
 ### Sample Hardhat project
 
-If you select _Create a sample project_ a simple project creation wizard will ask you some questions and create a project with the following structure:
+If you select _Create a JavaScript project_, a simple project creation wizard will ask you some questions and then a project with the following structure will be created:
```

Comment:
This paragraph should explain that it will come with the Hardhat toolbox, and explain what it is.

Reply:
Please check f1186757798e54479c971c17f4e4cac38b5c7b94